pkg/frontend/adminactions/adminactions.go

```diff
@@ -0,0 +1,119 @@
+package adminactions
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/dynamichelper"
+	"github.com/Azure/ARO-RP/pkg/env"
+	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/compute"
+	"github.com/Azure/ARO-RP/pkg/util/refreshable"
+	"github.com/Azure/ARO-RP/pkg/util/restconfig"
+)
+
+// Interface for adminactions
+type Interface interface {
+	KubeGet(oc *api.OpenShiftCluster, groupKind, namespace, name string) ([]byte, error)
```

Comment:
I prefer not having `kube` in the names here.

Reply:
Yes, but now admin.Get() is what exactly? the cluster/a resource or a kube object. i think it is needed to know what it is doing